### PR TITLE
Quartus utils

### PR DESCRIPTION
**Pull request by qarlosalberto**

### CODE REVIEW

File: packages/colibri/src/project_manager/multi_project_manager.ts [hunk 1]
```diff
@@ -255,13 +255,13 @@ export class Multi_project_manager {
...
     ////////////////////////////////////////////////////////////////////////////
 
-    async create_project_from_quartus(prj_path: string): Promise<Project_manager> {
-        const result = await get_project_info_from_quartus(this.get_config_global_config(), prj_path);
-        const prj = this.initialize_project(result.prj_name);
-        await prj.add_file_from_quartus(this.get_config_global_config(), prj_path, true);
-        prj.add_toplevel_path_from_entity(result.prj_top_entity);
-        return prj;
-    }
+    // async create_project_from_quartus(prj_path: string): Promise<Project_manager> {
```
Comment (asaezper) [inline packages/colibri/src/project_manager/multi_project_manager.ts:258]: is this done on purpose?

File: packages/colibri/src/project_manager/tool/quartus/quartusProjectManager.ts [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+        this.quartusProjectWatcher = chokidar.watch('file', {
+            usePolling: true, interval: 2000
+        });
+        this.quartusProjectWatcher.on('change', (_path, _stats) => {
+            this.refreshFromQuartusProject();
+        });
+    }
+
+    async refreshFromQuartusProject() {
```
Comment (asaezper) [inline packages/colibri/src/project_manager/tool/quartus/quartusProjectManager.ts:29]: it shouldn't be any specific function, just functions in Project_manager overwritten here

**Review by asaezper — APPROVED** [review]

File: packages/colibri/src/project_manager/multi_project_manager.ts (hunk 1 above)
Comment (qarlosalberto) [reply packages/colibri/src/project_manager/multi_project_manager.ts:258]: yes, I need to think how to do it in a next PR

File: packages/colibri/src/project_manager/tool/quartus/quartusProjectManager.ts (hunk 2 above)
Comment (qarlosalberto) [reply packages/colibri/src/project_manager/tool/quartus/quartusProjectManager.ts:29]: yes, I need to think how to do it in a next PR

Comment (qarlosalberto) [reply packages/colibri/src/project_manager/tool/quartus/quartusProjectManager.ts:29]: you are agree

**Pull request merged**